include/pmix_common.h.in

```diff
@@ -1478,6 +1478,9 @@ static inline void* pmix_calloc(size_t n, size_t m)
 #define PMIX_CHECK_RANK(a, b) \
     ((a) == (b) || (PMIX_RANK_WILDCARD == (a) || PMIX_RANK_WILDCARD == (b)))
 
+#define PMIX_NSPACE_INVALID(a) \
```

Comment:
unrelated? :)

Reply:
Yeah it is - just happened to be in there at the same time, and not worth a separate pass thru the CI 😄